app/src/main/res/values-fr/strings.xml

```diff
@@ -1,426 +1,399 @@
-<?xml version="1.0" encoding="utf-8"?>
-<resources>
+<resources xmlns:tools="http://schemas.android.com/tools">
     <string name="app_name">PDF Converter</string>
     <string name="title">Actions PDF</string>
-
     <string name="file_name_text">Nom du fichier</string>
-    <string name="select_images_text">Choisir Images</string>
-    <string name="password_protect_pdf_text">Password Protect PDF</string>
-    <string name="edit_images_text">Edit Images</string>
-    <string name="open_pdf_text">Ouvrir PDF</string>
+    <string name="select_images_text">Choisir des images</string>
+    <string name="password_protect_pdf_text">PDF protégé par mot de passe</string>
+    <string name="edit_images_text">Modifier des images</string>
+    <string name="open_pdf_text">Ouvrir un PDF</string>
     <string name="create_pdf_text">Créer PDF</string>
     <string name="details"><b>Détails</b></string>
-    <string name="search_hint">Name of file</string>
-    <string name="search">search</string>
-
+    <string name="search_hint">Nom de fichier</string>
+    <string name="search">recherche</string>
     <array name="items">
         <item>Ouvrir Fichier</item>
         <item>Supprimer Fichier</item>
         <item>Renommer Fichier</item>
         <item>Imprimer Fichier</item>
-        <item>Share File</item>
+        <item>Partager le fichier</item>
         <item>afficher les détails</item>
-        <item>Ajouter mot de passe</item>
+        <item>Ajouter un mot de passe</item>
         <item>Supprimer le mot de</item>
         <item>Faire pivoter les pages</item>
-        <item>Add Watermark</item>
+        <item>Ajouter une filigrane</item>
     </array>
-
-    <string name="please_wait">Patientez</string>
+    <string name="please_wait">Veuillez patienter</string>
     <string name="create_pdf">Accueil</string>
-    <string name="viewFiles">Voir Fichiers</string>
-
-    <string name="bundleKey">ImageUri</string>
-    <string name="whatsappToast">Sorry! Images could not be added. Please select the images from our **Select Images** option.</string>
```

Comment:
this is still deleted

Reply:
Hi, I put back all the untranslated strings in english which will be translated later :)
Regards,